spectral/scripts/rules.sh

```diff
@@ -7,7 +7,23 @@
 
 echo "fetching kas-fleet-manager from main" 
 wget -P .openapi https://raw.githubusercontent.com/redhat-developer/app-services-sdk-js/main/.openapi/kas-fleet-manager.yaml
+echo "fetching srs from main" 
+wget -P .openapi https://raw.githubusercontent/redhat-developer/app-services-sdk-js/main/.openapi/srs-fleet-manager.json
 
-## TODO add more apis
+# updates made to the following need to be refected in their respective .openapi files before the linting can pass
+# echo "fetching connector_mgmgt from main" 
+# wget -P .openapi https://raw.githubusercontent.com/redhat-developer/app-services-sdk-js/main/.openapi/connector_mgmt.yaml
+# echo "fetching smartevents from main" 
+# wget -P .openapi https://raw.githubusercontent.com/redhat-developer/app-services-sdk-js/main/.openapi/smartevents-mgmt.yaml
+# echo "fetching kafka-admin from main" 
+# wget -P .openapi https://raw.githubusercontent.com/redhat-developer/app-services-sdk-js/main/.openapi/kafka-admin-rest.yaml
```

Comment:
`https://raw.githubusercontent.com/redhat-developer/app-services-sdk-js/main/.openapi/` should be env var

Reply:
done